lib/rom/site/project.rb

```diff
@@ -8,6 +8,14 @@ class Project < Middleman::Docsite::Project
       def org
         'rom-rb'
       end
+
+      def latest_path
+        "/learn/#{name}/#{latest_version}"
+      end
+
+      def latest_version
+        versions.max_by { |version| version[:value].to_f }[:value]
```

Comment:
we could use `Gem::Version` here

Reply:
@ianks oh thanks that's a great suggestion. I'll be moving this method to middleman-docsite so I'll use `Gem::Version` there.